docs/releases/v2020.1.rst

```diff
@@ -0,0 +1,189 @@
+-  AVM FRITZ!Repeater 1200
+
+ipq806x-generic
+~~~~~~~~~~~~~~~
+
+-  Netgear R7800
+
+lantiq-xway
+~~~~~~~~~~~
+
+-  AVM FRITZ!Box 7312
+-  AVM FRITZ!Box 7320
+-  AVM FRITZ!Box 7330
+-  AVM FRITZ!Box 7330 SL
+
+lantiq-xrx200
+~~~~~~~~~~~~~
+
+-  AVM FRITZ!Box 7360 (v1, v2)
+-  AVM FRITZ!Box 7360 SL
+-  AVM FRITZ!Box 7362 SL
+-  AVM FRITZ!Box 7412
+
+mpc85xx-p1020
+~~~~~~~~~~~~~
+
+-  Enterasys WS-AP3710i
+-  OCEDO Panda
+
+ramips-mt7620
+~~~~~~~~~~~~~
+
+-  TP-Link Archer C2 (v1)
+-  TP-Link Archer C20 (v1)
+-  TP-Link Archer C20i
+-  TP-Link Archer C50 (v1)
+-  Xiaomi MiWifi Mini
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+-  Netgear EX6150 (v1)
+-  Netgear R6220
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+
+-  GL.iNet VIXMINI
+-  TP-Link TL-MR3020 (v3)
+-  TP-Link TL-WA801ND (v5)
+-  TP-Link TL-WR902AC (v3)
+
+Removed Hardware support
+------------------------
+
+-  ALFA Network Hornet-UB
+-  ALFA Network Tube2H
+-  ALFA Network N2
+-  ALFA Network N5
+
+Major changes
+-------------
+
+OpenWrt 19.07
+~~~~~~~~~~~~~
+
+Gluon v2020.1 is the first release to use OpenWrt 19.07. All targets
+therefore use Linux 4.14.166.
+
+batman-adv compat v14 removal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Support for the long deprecated compat 14 version of batman-adv has been
+dropped. Communities still using this version should migrate to batman-adv
+using the scheduled domain switch.
+
+IBSS wireless mesh removal
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Support for the IBSS wireless protocol has been dropped. Communities
+still using IBSS are suggested to migrate to 802.11s using the scheduled
+domain switch.
+
+Performance enhancements
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+We install zram-swap by default on ``ar71xx`` devices with 8MB of flash
+and 32MB of RAM.
+
+Target renaming
+~~~~~~~~~~~~~~~
+
+- The ``ipq40xx`` target was renamed to ``ipq40xx-generic``.
+- The ``ipq806x`` target was renamed to ``ipq806x-generic``.
+
+Status Page
+~~~~~~~~~~~
+
+- Gateway nexthop information has been added to the statuspage when batman-adv
+  is used. This includes its MAC address and prettyname as well as the interface
+  name towards the selected gateway.
+- The site name has been added to the statuspage. If the node is in a multidomain
+  setup it will also show the domain name.
+
+DECT button to enter config mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Many AVM devices don’t feature a separate RESET/WPS button, therefore
+starting this release we support entering the config mode via DECT buttons.
+
+Bugfixes
+--------
+
+Autoupdater aliases
+~~~~~~~~~~~~~~~~~~~
+
+We have added several new aliases for autoupdater compatibility on
+the following devices:
+
+- Ubiquiti UniFi AC LR
+- Raspberry Pi
+
+Site changes
+------------
+
+site.mk
+~~~~~~~
+
+-  The ``GLUON_WLAN_MESH`` variable can be dropped, as 802.11s is
+   the only supported wireless transport from now on.
+
+Internals
+---------
+
+x86 partition size
+~~~~~~~~~~~~~~~~~~
+
+The x86 partition size has been reduced to fit on disk with 128 MB
+capacity.
```

Comment:
This is a user-visible change, not an internal.

Reply:
Moved to major changes.